blog.html

```diff
@@ -0,0 +1,22 @@
+---
+layout: blog
+title: Blog Archive
+permalink: /blog
```

Comment:
Is this causing

```
jekyll 2.2.0 | Error:  Is a directory @ rb_sysopen - /home/vijaykumar/projects/pysangamam-web/_site/blog
```

Reply:
I had to change the permalink to /blog.html. But if this works for you, then I am OK with the changes.